js/app.js

```diff
@@ -363,16 +349,17 @@
 				});
 			}
 		},
+		addParticipantToCall: function(token, participant) {
+			return this.signaling.addParticipantToCall(token, participant);
```

Comment:
Okay and when a moderator kicks a user from a room we send a request to signaling and the room management then?

Basically need to do the same when adding users, because the data must be persistent?